### PR TITLE
Feature/co 73

### PR DESCRIPTION
**Pull request by DanielVelando**

### CODE REVIEW

File: frontend/src/components/DatePicker/DatePicker.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import "react-datepicker/dist/react-datepicker.css";
+import "./DatePicker.module.css";
+
+import React from "react";
+import ReactDatePicker from "react-datepicker";
+
+interface Props {
+  isClearable?: boolean;
+  onChange: (date: Date) => any;
```
Comment (objmj-itminds) [inline frontend/src/components/DatePicker/DatePicker.tsx:9]: use void instead of any.

File: frontend/src/components/DatePicker/DatePicker.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  selectedDate: Date | null;
+  showPopperArrow?: boolean;
+}
+
+const DatePicker = ({
```
Comment (objmj-itminds) [inline frontend/src/components/DatePicker/DatePicker.tsx:14]: Remember to use FC type for components; it is important to use if libraries require that specific JSX.Element type.

File: frontend/src/components/DatePicker/DatePicker.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import "react-datepicker/dist/react-datepicker.css";
+import "./DatePicker.module.css";
```
Comment (objmj-itminds) [inline frontend/src/components/DatePicker/DatePicker.tsx:2]: I don't think these styles are used correctly. Have you seen them work? 
If I am correct, you must specify an object and use the className attribute on the component.

File: frontend/src/components/DatePicker/DatePicker.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  selectedDate: Date | null;
+  showPopperArrow?: boolean;
+}
+
+const DatePicker = ({
+  selectedDate,
+  onChange,
+  isClearable = false,
+  showPopperArrow = false,
+  ...props
```
Comment (objmj-itminds) [inline frontend/src/components/DatePicker/DatePicker.tsx:19]: to get this, what you want to do is

```typescript
const DatePick: FC<Props & Partial<REACTDATEPICKERPROPTYPE>> ...
```
This way, you get the component fully typed while still using spread to get any other props.

File: frontend/src/components/OrderRefill/OrderRefillComp.tsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
+import "react-datepicker/dist/react-datepicker.css";
```
Comment (objmj-itminds) [inline frontend/src/components/OrderRefill/OrderRefillComp.tsx:1]: So why is this style import in this place and not inside the component itself?

File: frontend/src/pages/demo/co73orderrefill.tsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+import { IOrderRefillCommand, OrderRefillCommand } from "services/backend/nswagts";
+
+const DemoPage: NextPage = () => {
+  const toast = useToast();
+  const { awaitCallback } = useOffline();
+  const bg = useColorModeValue("gray.100", "gray.700");
+
+  const postOrderRefill = useCallback(
+    async (orderRefill: IOrderRefillCommand) => {
+      awaitCallback(async () => {
```
Comment (objmj-itminds) [inline frontend/src/pages/demo/co73orderrefill.tsx:18]: This does not need to work offline—no need to use this hook. Well thought tho, just not needed :)

**Review by objmj-itminds — CHANGES_REQUESTED** [review]
Just some minor adjustment. The most important thing is nailing the imports and styles of the DatePicker component; I feel like something is amiss.

File: frontend/src/components/OrderRefill/OrderRefillComp.tsx (hunk 5 above)
Comment (DanielVelando) [reply frontend/src/components/OrderRefill/OrderRefillComp.tsx:1]: My bad, forgot to remove it

**Review by objmj-itminds — APPROVED** [review]

**Pull request merged**